tendrl/commons/objects/memory/__init__.py

```diff
@@ -1,56 +1,23 @@
-from tendrl.commons.event import Event
-from tendrl.commons.message import ExceptionMessage
+import psutil
```

Comment:
you need to separate imports by 2 blank lines
- python stdlibs
- third party (like psutil)
- imports from tendrl

Reply:
ack